physionet-django/user/forms.py

```diff
@@ -247,8 +247,10 @@ class Meta:
             'organization_name', 'job_title', 'city', 'state_province',
             'zip_code', 'country', 'webpage')
         help_texts = {
-            'first_names': "First name(s).",
-            'last_name': "Last (family) name.",
+            'first_names': """First name(s). Your profile first name by default. This 
```

Comment:
Perhaps just "Your first name(s)" would be clearer? Not sure I'd know what was meant by "Your profile first name by default".

Reply:
I can change it, seems more concise too